cmd/deployment.go

```diff
@@ -57,40 +57,56 @@ var (
 	sshKey                string
 	knowHosts             string
 	CreateExample         = `
-# Create new deployment with Celery executor (default: celery without params).
+	# Create new deployment with Celery executor (default: celery without params).
```

Comment:
Oh sorry, I meant the other way around like:
```go
    CreateExample = `
# Create new deployment ...
$ astro deployment create
`
```

Because otherwise the command's usage will render like:
```bash
astro deploymen team create --help
# usage...
       # Create new deployment with Local executor
    $ astro deployment create new-deployment-name-local --executor=local
```

Reply:
No problem, have corrected it. PTAL